src/Stryker.Core/Stryker.Core/TestRunners/VsTest/VsTestRunner.cs

```diff
@@ -308,7 +308,7 @@ private void ParseResultsForCoverage(IEnumerable<TestResult> testResults, IEnume
             // since we analyze mutant coverage, mutants are assumed as not covered
             var seenTestCases = new HashSet<Guid>();
             var dynamicTestCases = new HashSet<Guid>();
-            var mutantCount = mutants.Max(m => m.Id) + 1;
+            var mutantCount = mutants.Any() ? mutants.Max(m => m.Id) + 1 : 0;
```

Comment:
How does this work? This selects the highest mutant Id right? Not the mutantcount. I'm suddenly doubting if this is correct